extensions/wikia/FeedsAndPosts/JustUpdatedAPIProxy.class.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class JustUpdatedAPIProxy {
+
+	const LIMIT = 4;
+
+	private function requestAPI($params) {
+		$api = new \ApiMain(new \FauxRequest($params));
+		$api->execute();
+
+		return $api->GetResultData();
+	}
+
+	private function getRecentChanges() {
+		$response = $this->requestAPI( [
+			'action' => 'query',
+			'list' => 'recentchanges',
+			'rcprop' => 'title',
```

Comment:
what about this requirement? `Nice-to-have: Filtered by those with a before and after character count data of X`
To do that just add `|size` here. The size represents number of bytes changed (can be negative for deletions)